### DISCUSSION TITLE
Alternative for combineLatest other than forkJoin

### DISCUSSION THREAD

**Discussion by Goaman** [ReactiveX/rxjs#6725] [Q&A] [answered]
I was trying to use `combineLatest(...array)` but it seems that the signature is being deprecated.
I've followed the link that describe the recommended way: https://rxjs.dev/deprecations/array-argument#array-arguments

The problem is that the behavior of  `combineLatest(...array)` vs  `forkJoin(array)` is different:
`forkJoin` emits only when all inner observables have completed.

In my case, I want it to emit even without the inner observables being completed.
How am I supposed to handle this case ?

Shouldn't this case be documented on the url I mentioned ?

**Comment by kwonoj** [comment, marked as answer]
That documentation means you can apply those signatures for both of operators, not meant you need to replace operator to forkjoin itself.

**Reply by Goaman** [reply]
Ok, indeed I misunderstood the point of the documentation page.
Thank you for the clarification.
